src/components/ha-onboarding-banner.ts

```diff
@@ -0,0 +1,100 @@
+import { LitElement, html, css, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import type { HomeAssistant } from "../types";
+import { navigate } from "../common/navigate";
+import "./ha-icon";
+import "./ha-icon-button";
+
+@customElement("ha-onboarding-banner")
+export class HaOnboardingBanner extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _dismissed = false;
+
+  private _navigate(ev: Event) {
+    ev.preventDefault();
+    if (this.hass && this.hass.connected) {
+      navigate("/config/devices/dashboard");
+    }
+  }
+
+  private _dismiss() {
+    this._dismissed = true;
+  }
+
+  render() {
+    const devices = this.hass.devices ? Object.values(this.hass.devices) : [];
+    const physicalDevices = devices.filter(
+      (device) =>
+        device.entry_type !== "service" && device.connections.length > 0
+    );
+    const hasPhysicalDevices = physicalDevices.length > 0;
```

Comment:
Why do this work, even if the user might have already dismissed the banner and we don't need it?